python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -154,6 +155,7 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
         normalizeResiduals = lsstDebug.Info(__name__).normalizeResiduals
                                                             # Normalise residuals by object amplitude
 
+        debugLog = pexLog.Debug("meas.algorithms.psfDeterminer")
```

Comment:
Why did you add this debug log? This task already has a log attribute and the existing code used it.